utils/concatenate.cwl

```diff
@@ -24,7 +26,11 @@ stdout: $(inputs.outputFileName)$(inputs.postfix)
 outputs:
   - id: result
     type: stdout
-    format: ${if ("format" in inputs.files[0]) return inputs.files[0].format; else return 'undefined'}
+    format: ${if (typeof(inputs.files[0]) == 'string') return 'underfined';
```

Comment:
Typo `underfined`. Also shouldn't that be `undefined` with no quotes?

Reply:
It seems working with quotes :thinking_face: